vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -1,7 +1,7 @@
 import logging
```

Comment:
Putting comment here because I can't put it elsewhere... Can we get rid of the `def _update_theme` now?
1. What happens if you remove the function altogether? Do we get flickering or is it ok?
2. Failing that, let's put it in `__call__` directly - the only reason it was done as a separate function before was for reuse, but now it's only used once

Reply:
Removing theme setting in `Graph.__call__` (and adding `"graph.figure"` as a clientside_callback `_update_graph_theme` dcc.Input) causes graphs to flicker. However, let me do a little PR by moving the `_update_theme` directly to `Graph.__call__`.

Reply:
Cool, thank you for checking and raising the PR! 👍